src/Features/Core/Portable/CodeRefactorings/MoveType/AbstractMoveTypeService.State.cs

```diff
@@ -18,58 +14,22 @@ private sealed class State
     {
         public SemanticDocument SemanticDocument { get; }
 
-        public TTypeDeclarationSyntax TypeNode { get; set; }
-        public string TypeName { get; set; }
-        public string DocumentNameWithoutExtension { get; set; }
-        public bool IsDocumentNameAValidIdentifier { get; set; }
+        public TTypeDeclarationSyntax TypeNode { get; }
```

Comment:
reduced this to the data used in many different locations.  removed almost everything else.